tasks/82_bouncing-balls/bouncing-balls.js

```diff
@@ -0,0 +1,15 @@
+export const bouncingBall = (height, bounce, window) => {
+  if (height < 0 || bounce >= 1 || bounce <= 0 || window > height) {
+    return -1;
+  }
+  let seenByMum = 0;
+  while (height > window) {
```

Comment:
FYI there is math solution with `Math.floor(Math.log(window / height) / Math.log(bounce)) * 2 + 1;`
Math rule :D